### ISSUE TITLE
Allow answer 'Both fountain and bottle refill' for drinking water type quest

### ISSUE THREAD

**Issue by dbdean** [streetcomplete/StreetComplete#4492]
**Use case**
Around my area in NSW most drinking water comes from a fountain that also has a bottle refill spout. Like this one:

![image](https://user-images.githubusercontent.com/12725310/195488449-fdcda871-b181-4740-a28e-431a1b15d29a.png)
(My photo, and happy to provide to the project as an example image)

**Proposed Solution**

I would appreciate an answer on  the drinking water type quest that indicates that is is both a fountain and can refill bottles.

I believe the correct tagging for this would be `fountain`=`bubbler` and `bottle`=`yes`.

**Comment by westnordost** [comment]
Assigning to @matkoniecz because he is the author of that quest

**Comment by mnalis** [comment]
Generally, `bottle=*` is problematic as it is [unclear](https://wiki.openstreetmap.org/wiki/Key:bottle#Unclear_use). While in this specific case it obviously `bottle=yes` by all standards, in _many other cases_ it isn't, and **adding more usage/legitimacy to obviously problematic tag is probably not a good idea**. (as data consumers have no way to know which of those unclear definitions of `bottle=yes` the particular amenity/mapper subscribed to).

In SC quest, choosing `refilling bottles ONLY` marks as [fountain=bottle_refill](https://wiki.openstreetmap.org/wiki/Key:fountain) instead (which would overlap with `bottle=only` if one were to accept legitimacy of [bottle=*](https://wiki.openstreetmap.org/wiki/Key:bottle) tag). 

As to how to actually map it, it depends:
- one could map it is generic `other water fountain`, i.e. [fountain=drinking](https://wiki.openstreetmap.org/wiki/Tag:fountain%3Ddrinking) and be done with it (that is exactly why that answer is offered, after all!)
- one could leave a note with picture
- one dedicated to such precision could micromap as two very close but separate `amenity=drinking_water`, one [fountain=bubbler](https://wiki.openstreetmap.org/wiki/Tag:fountain%3Dbubbler) and other [man_made=water_tap](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dwater_tap)+[handle=button](https://wiki.openstreetmap.org/wiki/Key:handle) (that currently would involve using some other editor)
- one can hide the quest, or disable the confusing quest completely if such overthinking hurts and happens too often
- one could [discuss the tagging](https://wiki.openstreetmap.org/wiki/Tagging_mailing_list) and implement a new value for `fountain=*` which defines such a combo for bubbler+tap

**Comment by matkoniecz** [comment]
I think it is clear that current tagging is fine, and it is request for extra feature.

I would at least wait with it as there are already huge discussions on tagging (triggered by my research for this quest) and several proposals being active.

I am not entirely sure about `bottle=yes` tag as discussed at https://wiki.openstreetmap.org/wiki/Key:bottle#Unclear_use - but at the same time it would be nice to tag it somehow.

I think I will close it, mostly because I literally spend more than 20h on researching, discussing and trying to improve drinking-water related tagging and I am not ready for more.

And current `bottle=yes` tag is a bit problematic.

I would probably do

> one (dedicated to such precision) could micromap as two very close but separate amenity=drinking_water, one [fountain=bubbler](https://wiki.openstreetmap.org/wiki/Tag:fountain%3Dbubbler) and other [man_made=water_tap](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dwater_tap)+[handle=button](https://wiki.openstreetmap.org/wiki/Key:handle) (that currently would involve using some other editor)

with Vespucci if I would encounter it

**Comment by matkoniecz** [comment]
If someone thinks that `bottle=yes` works well or that there is a good solution here: feel free to discuss it at OSM Wiki or tagging mailing list or https://community.openstreetmap.org/ - and let me know here about conclusion if there is something we can use.

Still, it would be tricky as it is basically 2 in 1.

**Comment by westnordost** [comment]
So I understand that @matkoniecz is not categorically against such a suggestion but requests that this is first discussed in the forum?

**Comment by matkoniecz** [comment]
Not against it, but ideally wiki would be changed first - and if it is right that this tag is bad then it should not be used in SC.

I am not planning to spend time on discussing it as this topic is not really tractable (as far as my entertainment time spend on tag cleanup goes I have several more promising targets).

**Comment by mnalis** [comment]
> Not against it, but ideally wiki would be changed first - and if it is right that this tag is bad then it should not be used in SC.

I agree that is should first be discussed on [tagging ML](https://wiki.openstreetmap.org/wiki/Tagging_mailing_list) and/or [community forum](https://community.openstreetmap.org/c/general/tagging/70), and then wiki changed according to consensus. if you decide to do it, please link any such discussion to this github issue @mxxcon so interested parties may participate.

If the community consensus is that it makes sense to tag `bottle=*`, then it can be reconsidered for SC quest too, I agree.

> I think people are being overly pedantic about the wording of the wiki. Refusing to have a toggle for bottle=yes because of an argument over the definition of "easily" on the wiki 

There are several different problems. Did you read https://wiki.openstreetmap.org/wiki/Key:bottle and https://wiki.openstreetmap.org/wiki/Talk:Key:bottle in addition to this ticket? When bringing it up to community discussion (if you decide to do so), you should address **all** of the issues mentioned, not just one.

> ultimately makes the map less useful.

_"Citation needed"_. What **is actual usefulness** of `bottle=yes`? Are there any data consumers using it? Or at least any anecdotal situation where you personally wished _"man, my life would've been so much easier that time if I just knew in advance whether that water fountain was `bottle=yes` or just regular fountain where I could fill my bottle too"_? Such cases too should be mentioned in that ML/forum discussion as they'd provide useful datapoints. 

> People understand what is the implied meaning of bottle-filling part of a water fountain is.

Do they? I don't, for example - despite being OSM mapper very interested in water sources and involved with that tag discussions and wiki documentation. In fact, there are at least two different opinions what that tag is supposed to mean, see wiki :shrug: Have we got any statistics about user understanding of tag `bottle=*`? It seemed to me (at best) that it is highly country-specific (but more likely conflicting between different groups even in the same country). For example, in Croatia, the thing you show in the [middle of the picture](https://github.com/streetcomplete/StreetComplete/issues/4492#issuecomment-1490493951) does not really exist (or exists extremely rarely).

Also, that quest is not **only** about drinking fountains, but all sources of drinking water. See other pictures of answers of that quest: springs, pumps, wells, ... Virtually **all of them** can be used to fill _most_ bottles one way or another (some easier, some harder, some much harder).

> If somebody has an overly large bottle, instead of standing it on a shelf, they'll just hold it at an angle.

Yes. Which is exactly what they would be doing too **if the thing in middle of your picture was missing**, and only regular 60-degree-angle jets were available - they'd hold bottle at the angle so jet terminates in bottle, and they'd **fill the bottle that way**. As you make an argument that the fact that one is slightly more "easy" than another doesn't matter, one would conclude that both versions of water fountain (i.e. one with _"just water jets"_ **and** one with _"thingy in the middle"_) should be marked with `bottle=yes`, right? 

In other words, **do you  envision situation** where `bottle=no` should be tagged? Because, if `bottle=yes` should be tagged on virtually any answer (as in all those cases currently shown one could fill a bottle with more or less convenience), then in my opinion it doesn't make much sense to tag it at all.

> In the grand scheme of things I don't see how having that tag would cause more harm and confusion than not having that tag at all.

In my opinion, it creates huge mental load about what to choose and confuses users (which might  lead to notes being left instead of quest answered, or even worse - in confusing quest being disabled altogether by users) -- and (most importantly IMHO) produces **no usable information** for data consumers as a result. Thus, it is better avoided.

**Comment by mxxcon** [comment]
Yes I read the wiki and talk pages before posting my previous message.
I'm not going to use mailing list for anything, so that's out of the question.
I'm also not going to post on the community forum about this tag. I retract my request to have that option added and lost all interest and desire to be involved in this topic. By the previous message it's obvious that people are more interested in navel-gazing arguments on the meaning of "quickly" and "easily" and ask questions for the sake of asking those questions rather than adding value to the map.
I understand discussing important tags with truly ambitious meanings(hello gender neutral bathrooms), but not bottle=yes/no.
I spent enough time on OSM as it is to spend any more talking about this tag. I'd rather be editing.

 ridiculous pedantry like this is why overturemaps was created and why

**Comment by matkoniecz** [comment]
> Ridiculous pedantry like this is why overturemaps was created 

not really (it was also not created due to 0.01% of tagged surfaces using unusual not documented values)
